### ISSUE TITLE
'Show tabbing' button doesn't focus radio buttons, partial keyboard trap when Tabbing manually

### ISSUE THREAD

**Issue by aorlova2909** [designsystemau/gold-design-system-site#15]
# Bug report

- [x] I’ve read and understood the [Contributing guidelines](https://github.com/designsystemau/design-system-site/blob/master/CONTRIBUTING.md) and have done my best effort to follow them.
- [x] I’ve read and agree to the [Code of Conduct](https://github.com/designsystemau/design-system-site/blob/master/CODE_OF_CONDUCT.md).
- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

1. When 'Show tabbing' button is activated, focus goes only through the checkboxes but not the radio buttons.
2. There seems to be a partial focus trap in the 1st checkbox, i.e. the keyboard user can't press Tab to focus the next checkbox, but can press Shift-Tab to navigate back.

## What I expected to happen

1. When 'Show tabbing' button is activated, Tab focus should go through both - checkboxes and radio buttons.
2. The keyboard user should be able to move Tab focus to the 2nd checkbox

## Reproducing

- Website URL: https://gold.designsystemau.org/components/control-input/accessibility/
- Browser: Chrome
- Operating system: Windows

Steps to reproduce:

1. Open https://gold.designsystemau.org/components/control-input/accessibility/
2. In 'Keyboard accessibility' section activate 'Show tabbing' and observe behavior
3. Press Tab key to focus 1st checkbox in the same section and press Tab again trying to move focus to the next checkbox

## Attachments

https://user-images.githubusercontent.com/89005205/139833443-5deefc4f-5869-4e0b-a7f1-04e3e33142d1.mp4

**Comment by Zubiii** [comment]
This issue has been resolved in pull request. Thanks.

**Comment by dominikwilkowski** [comment]
fixed via #19
